### DISCUSSION TITLE
조직과 프로젝트 이름을 바꿔봤습니다.

### DISCUSSION THREAD

**Discussion by mascari4615** [wrchat/Woodon#38] [General]
버전을 `v0.1.0`으로 올리면서 조직과 프로젝트 이름을 바꿔봤습니다.

- 조직 : `WakVRC` -> `WRChat`
- 프로젝트 : `MUdons` -> `Woodon`

기존 개인 프로젝트를 공동 프로젝트로 변환하게 되면서, 여러 곳에서 제 향을 지우는 과정에 있습니다.
그 중 프로젝트 이름은 특히 빨리 바꿔야겠다는 생각을 해서, 다른 분들의 의견을 듣지 않고 무작정 바꿔봤습니다.

어떤 것 같나요?

기존 조직 이름을 `WakVRC`로 정했을 때와 마찬가지로, 이 조직/프로젝트 이름은 당장 임시로 사용하게 될 이름이며, 언젠가 참여 인원이 꽤 생긴다면 의견을 모아 새로운 이름을 정할 것 같습니다.

**Comment by Listing2** [comment]
특히 프로젝트 이름이 직관적이어서 좋은 것 같스빈다~~
